### PR TITLE
WL-2826 : Ability of survey owner to change owner of survey

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: api/src/java/org/sakaiproject/evaluation/logic/EvalEvaluationService.java [hunk 1]
```diff
@@ -112,6 +112,17 @@
     public String updateEvaluationState(Long evaluationId);
 
     /**
+     * This will update the
+     * owner of the evaluation
+     *
+     * @param evaluationId the id of an EvalEvaluation object
+     * @param userId the internal user id (not username)
+     * @return an EvalEvaluation
```
Comment (buckett) [inline api/src/java/org/sakaiproject/evaluation/logic/EvalEvaluationService.java:120]: Might be worth documenting the IllegalArgumentException when evaluationId doesn't exist.

Comment (nicholaswilson100) [inline api/src/java/org/sakaiproject/evaluation/logic/EvalEvaluationService.java:120]: Ok

File: tool/src/java/org/sakaiproject/evaluation/tool/producers/ControlEvaluationsProducer.java [hunk 2]
```diff
@@ -348,10 +357,15 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
             UIOutput.make(evaluationRow, "active-eval-startdate", df.format(evaluation.getStartDate()));
             // TODO add support for evals that do not close - summary.label.nevercloses
             UIOutput.make(evaluationRow, "active-eval-duedate", df.format(evaluation.getSafeDueDate()));
+             EvalUser owner = commonLogic.getEvalUserById( evaluation.getOwner() );
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/evaluation/tool/producers/ControlEvaluationsProducer.java]: Indentation doesn't look right for most of the changes in this file.

File: tool/src/java/org/sakaiproject/evaluation/tool/producers/ChownEvaluationProducer.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This page allows the user to chnage owner of evaluations
+ * 
+ * @author Nick Wilson
+ */
+public class ChownEvaluationProducer implements ViewComponentProducer, NavigationCaseReporter, ViewParamsReporter {
+
```
Comment (buckett) [inline tool/src/java/org/sakaiproject/evaluation/tool/producers/ChownEvaluationProducer.java:38]: Evals tool seems to use spaces in most Producers as indentation.

**Comment by buckett** [comment]
Nice work.

**Comment by nicholaswilson100** [comment]
ok

**Pull request merged**